hiss/customauth/tests/view_tests/verify.py

```diff
@@ -33,14 +45,23 @@ def test_signup_creates_inactive_user(self):
 
     def test_signup_sends_email(self):
         self.client.post(reverse_lazy("customauth:signup"), self.fields)
+        User.objects.get(email=self.fields['email'])
         self.assertEqual(len(mail.outbox), 1)
+        body, _ = mail.outbox[0].alternatives[0]
+        url, _, _ = re.findall(URL_REGEX, body)[0]
+
+        user = User.objects.get(email=self.email)
+        self.assertTrue(self.isValidLink(user, url))
+
 
     def test_signup_sends_valid_confirmation_link(self):
         self.client.post(reverse_lazy("customauth:signup"), self.fields)
+        User.objects.get(email=self.fields['email'])
         self.assertEqual(len(mail.outbox), 1)
         body, _ = mail.outbox[0].alternatives[0]
         url, _, _ = re.findall(URL_REGEX, body)[0]
 
-        self.client.get(url)
+
         user = User.objects.get(email=self.email)
-        self.assertTrue(user.is_active)
+        # self.assertTrue(user.is_active)
```

Comment:
Please uncomment this, it's important that the user become activating after clicking the confirmation link.